/users3/uccibit/alexsayal/BIDS-BRAINPLAYBACK-TASK2/code/mvpa/mvpa-lsa-spacenet.py:100: UserWarning: Brain mask is bigger than the volume of a standard human brain. This object is probably not tuned to be used on such data.
  decoder.fit(X_train, y_train)
[Parallel(n_jobs=30)]: Using backend LokyBackend with 30 concurrent workers.
[Parallel(n_jobs=30)]: Done   2 out of  12 | elapsed:  9.4min remaining: 47.1min
.........[Parallel(n_jobs=30)]: Done   9 out of  12 | elapsed: 12.9min remaining:  4.3min
........[Parallel(n_jobs=30)]: Done  12 out of  12 | elapsed: 15.3min finished
Traceback (most recent call last):
  File "/users3/uccibit/alexsayal/BIDS-BRAINPLAYBACK-TASK2/code/mvpa/mvpa-lsa-spacenet.py", line 110, in <module>
    results1 = pd.DataFrame({'accuracy': accuracy})
               ^^
NameError: name 'pd' is not defined. Did you mean: 'id'?
.........................................................................................